### ISSUE TITLE
Use personal pronoun consistently

### ISSUE THREAD

**Issue by agryman** [moorepants/learn-multibody-dynamics#89]
A minor nit: In the following text from "Introduction", you switch from "I" to "we". 

```
With the goal of teaching through computation, it means I need to also choose a programming language. There are many programming languages well suited to multibody dynamics computation, but we select [Python](http://www.python.org/) for several reasons:
```

The book is singly-authored by you, so stick to "I".

If you agree with this choice, I'll go ahead and make this change going forward, along with other minor typos, and submit a PR.

**Comment by moorepants** [comment]
Yes, that one should be "I". There may be places were I do want it to be we, but I'd have to take a full pass again to see. I'll review a PR with the changes if you make it. I think I may have jumped between I and we a fair amount though :(

**Comment by agryman** [comment]
Jason,

OK, I'm proofreading from the start. I'll submit several smallish PRs.

-- Arthur

**Comment by agryman** [comment]
I'll fix this in my next PR.
